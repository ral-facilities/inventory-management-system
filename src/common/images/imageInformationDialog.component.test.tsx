import { act, screen } from '@testing-library/react';
import ImagesJSON from '../../mocks/Images.json';
import { renderComponentWithRouterProvider } from '../../testUtils';
import ImageInformationDialog, {
  ImageInformationDialogProps,
} from './imageInformationDialog.component';

describe('Image Information dialog Component', () => {
  let props: ImageInformationDialogProps;
  const onClose = vi.fn();

  const createView = () => {
    return renderComponentWithRouterProvider(
      <ImageInformationDialog {...props} />
    );
  };

  beforeEach(() => {
    props = {
      open: true,
      image: ImagesJSON[0],
      onClose: onClose,
    };
  });

  afterEach(() => {
    vi.clearAllMocks();
  });
  it('renders dialog correctly', async () => {
    let baseElement;
    await act(async () => {
      baseElement = createView().baseElement;
    });

    expect(screen.getByText('logo1.png')).toBeInTheDocument();
    expect(baseElement).toMatchSnapshot();
  });
});
