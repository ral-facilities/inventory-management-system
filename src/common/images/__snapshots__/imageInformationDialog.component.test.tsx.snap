// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Image Information dialog Component > renders dialog correctly 1`] = `
<body
  style="padding-right: 1024px; overflow: hidden;"
>
  <div
    aria-hidden="true"
  />
  <div
    class="MuiDialog-root MuiModal-root css-1r9csp7-MuiModal-root-MuiDialog-root"
    role="presentation"
  >
    <div
      aria-hidden="true"
      class="MuiBackdrop-root MuiModal-backdrop css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    />
    <div
      data-testid="sentinelStart"
      tabindex="0"
    />
    <div
      class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
      role="presentation"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      tabindex="-1"
    >
      <div
        aria-labelledby=":r0:"
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm MuiDialog-paperFullWidth css-tlc64q-MuiPaper-root-MuiDialog-paper"
        role="dialog"
      >
        <h2
          class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-bdhsul-MuiTypography-root-MuiDialogTitle-root"
          id=":r0:"
        >
          Image Information
        </h2>
        <div
          class="MuiDialogContent-root css-ypiqx9-MuiDialogContent-root"
        >
          <div
            class="MuiBox-root css-0"
          >
            <div
              class="MuiBox-root css-1og1qar"
            >
              <img
                alt="test"
                class="MuiBox-root css-0"
                src="data:image/webp;base64,UklGRrAdAABXRUJQVlA4WAoAAAAQAAAAvwAAvwAAQUxQSLYRAAABoEFbm95sG7wtWKlFg6VKKBap1YK1UqUWqzwWDRZjpVqlUgoGI6VQJDIwa2Xm/pnpvB9+KyIkSrIT1FKENfLUew4ghJD9AfU/TQcQx+8JKIvd+fef0ljmvyTAwvh3SmBXXf+apj6VKfyGMIV1W5yb7vns2mu5scq/kfN2NPhYhva4DV+AcMK+fs+GOKauXNM/oO2elqR2D0T4mIRl7GLM6sFBrUkKr52C6PPVUjXU4TIeb6m9IaRWPLQFTTFAdC5rYdy5xRpTYDtwm62Ug2nlmyphlSbupO60xrdZu94AQXLxMsYHfxOxgKopum4xpyw3+3PeWtYRGl1MwT8piDYXVDNTtzHKt/JMoKDWVhPApsFkCiLNMJDkxs/rNUwS7k/7kM9TA6PZ++b+0bZdrykwr2jPF6Mx3WaXWCjb/PJc2G6wnNckrviGg2jM0JRbF1X6R2E6KogyJ6Mr2swFCSg9f6xCF24VyEr3A4dvbnMHH0JZLJjXJ9LziSCdK2wwgsO56DhlPq6FkhmL0771lgBuAdCfdaEgRuWNm+XKsHJqcNfZn+NmD7bXPHB2fmCqBUMkqD/M7B6hgMow54Hn5JDMX0y9fFIFNLD9yJQ2yEhk6/aLkE2bGK+vRlPzgkypvq5CfJtAgaqYSn3t1hKveU3HefPLEW3bBKSDctKStYNNKOc+ckxzB0Yx9FQjbG10AmozIUqd17RSjnRqHSyg4GG0pkrTVfNvnH7FFpvk+KupFPigS+50Cz/Bjiqbnq7M61ojAT4Os6QKYn2rFloV5Be2kLXUr99/9iWBjPu1QcO0ThGfNK7Yp8ETYEY182K+6W42C1HaJQpCfR3nVSI7no5o81Tgb56hu9xRZqYwmFoFqI0PotfEJ0MwQhbWhUokguWgQmDukHTxGV3r10N0qACppF5BvcMxx/3lD8knOklnE3JGA2ew4PJnEwjwGckICiKzWgIPSHaw5tIDQuGtkHyT2GSHxcezIrATSWlUMLglHm+2sUluDDHthEJ6oCUPhdax3YNPnL/gExYohhuuu+RYsn9bQGX4YdwPg+Hd/zcIqHTiHxvXD+k/QfQqQZ4eJfPOkCgIe/EP3wIKPo6txf3ZumMgYPvMabQv2gfh02Fdm/pOYWjDvmgTm2Q63LwJ6o+qqsYanAtRiQrCzpvx2i1h/DuU/wLeRpNrrqB2S6R240lBgAkH56cLeUBJs71e7DmI1HUGA5LaU0BtJ6KW+rXtWCjTxht2i+QRpdcMeyR8307U0bjWETuJauflfvKg6MUmXZh1HxAbK9oZ4bnRLMy664PkL7QEXNA4625f1zg6Dgr44QB/uH6+6djW3QLmzpcTwm2hzUUB63YgLX8PSTG6Mj45kLT9PPvUDzL/qif1PSY+yAtm7yfShd9RLIDme+mw62waUZ7DEL0rPtl8EeuaFxT5ocibpgy2aWaP3c4+l263kC9Un7ZMLFzgpSH8lhEeH9aj4nInb7WDG5ZhmuZFo4bWyzJP00CGKJQAJHbO70p1jdhOdyQDcHRhm1fX5tGPJugx9s+2rvIt+7pkRFJHKESvXFKFN63S4tw8nfaUhAJoHocWwZuHZ3MuUkTDsa4j3HcQzH2lxZEe6o541t1CD3WIoM5DVx+cW4+obXQWo2yx5fuXntp+odr3ZwcLfBnu57RFMjpadEqPG4Xsij8/WAYEsUdsZ4cFOXX7PaupQ4LGGzA3sEdU4lDYVr35h453lbrtFc3929Nj9mkubkB+P+7N7XqqjoeyKIo8X+8rD8fqdKmbtnt93OE9QAeaH6etTVHk5HC3JRKGREejfuAB/kzhQcH0eVTbgu6Y2IJf35/dRrKKeKZ6fLanE+GkmTaonzsxqQ4AEgbqQxNGuVRV+xzFI6gD49tkVvlhhsPDSDQ80HbXQ2Y5tbgF3ekuiGuD8oooSMtr1/NzL8X2eQRb8KN3n3pLT8RofNZlCrRFH9JaAuuW4tdWaVk/vjKGa/lwTtb0C+06sprWhAvSBef+a32UvLEdYnOT8s4AQrv94g6hNUf2WzsxtYG1y2grSlBCmV7Aesk09ioFKyKX17aAchkbI2E8Xe0NIXO1anyJ7u6GXxWDykgvU0Be1AtJ58INg2m6hS2wN5xsYSgtf68aW/I6pyqZebvCWY9SHvFwki5GCyr7g2RUKq2e2imglU8RJoFKGh6x6a+phURU0ow2EcC1VPFS4SchfHaoZOOaWgdJevlQCkLQblbFv/RAP6Xd8DgHkrssZpwv7jNifw4CK2XXY7xjwsccHTDavJ1oxf7GkUphQhPX8vHqhrUj7zd1yQbSFgrkfwMRpJygdlNAr4vx6Onct70MhquHdiXhGfSBYbo13lfA8H6Tl85aRD7lIBiQJKgSeFSt0dROGH3dMdChsjVBVsXg5bZi4g2sdnQ4i4PXKPLa2Y7jtfTUvsfvd3y3lfPVCiYfTbio6OtKAYf/2BumNiol/xjOK0AK31hJ2GaM/3dHgKwFuXgs5ET1KAQUzgTH3KbeaHLypRVnFgkTelXT6JYalAjfi14ZP+hnuPp98oulR6pA/MhrlT1n+7AK1Aud/vgKYccAbWmKsNEP18rZC5X2tZN2wDMTn64Uh64kMB2cRaUrRyFdlTJhq3+iyr2QJGePvp0uBje17QPSQO8/pLHKjeZAd5uejDhNPEyRmUy1Aq8wiZct9Tc5drnRsqgKpw3pjMkSQ7TE5CkLXIWtB2G1fVEsbPcQWhL2ucbRxHLE0hHnToFwQJAUnhiEVGf9EFVS+FAm+5MWeKZeVD452dM3zx0rBXaHWiw8xHTmgfXga85S6cNFZVL5IMriHRCmq5FTq096cRxuPIert1lSUulCQkwmYzw3XHza1Zky+TWFTzowIJNJwiGI95SYoPhwWyNf+wga1mMj8GMlwIknTPv7fFMleCEoGJhKCLOvT6Cft6TYxZv3sVPAbsezvSJcVIQTGuBKgXEPEECUO0fQU5hg8uVOgi9LcCJoByCNLBziE2M/36M4e3zt65kdP/3lwXCwjSDZK/W+R9BkD6OJc6AYGObv3KjuLwOIT3ptA7kIDDbmGwY6Jjm040innKSzh9ABmYKXULEKAfudRn8AarfrFEwK+sx3Cf6QDfPSP++QBIxN2rBSBZN0wXKKQU5YdBZMjkgmV0LF5LCDGzFSSWo0xIV4P70JSZsLIHghefs4XfjBgV5xsPOIWMioBWmUWMDN/heSj8CU2isI1u+x9Eo07Yec+NE5aLALqB05zqCOHML16DG3HLlmbPzlpkDWBrJoBf+VT6WAHDuDPcz6d03P23XeR0EzI85UtE7AXud0c7619E6Bv7Q0VW1PpMDoP22KrGR64yJAmSanKWq4peO7A+wacCeBJesnF8E5sN1ObU6gNRh1wuPSCj1UvxH2dc+Htd3QZ5e4gmr1zSM9ygsjtqb1j+HuJkJcuYV5wGXTyOPqffXsazQ3yIsiE7V1LoeI/xq/4hC71ixSWRwrU+YFHT4YW+5FWR0ot6qzRGEXDLRfTYGv0a5NdTZa2ATwkqbFCToVul6r0mjCdvTIt7VSfDxnHZk+GZNkX/qygUm8dkreEseTRfICsdtmhGQK4DfL3mR+CIC5AeqEcgWZAion3esXqcW31s6doGgoKi1iIhlsK0XV4VGBMF80QWUhW4Drb5pi2Rdc4meokt706RRwLKsJ7b8Sb5oqxYLrMOo+gN9Y5fR+CMoCeM56iFK6DIFmRirC/GOG8JIHuLl96fO3TvhcEKAOpN+et4sdtFln6Bp5OFmWGrp2jW5TnmyJQKER7G009tcJOCTHdpjdsLT7UbBrTLrtn0p57pukjWaqo3E4SFOlmcKlzRz0tM1C7nWo3GPCc2BAusvzXQoIu2DKJIxtZkuPw541i2Hq4sxkQRdhN4l0l1vS7PSKzCcgerpozb8iIyH4xC7cFdD/MQBfuvft75qrh2EXdOPTRfA6JZh8Tl6jFaynyQfoX+W7behOouTDrpZUL4e6cNvTpzAfeOXb5oTHiZieeT9ZGXbfvGD9aPTlHd3ffHlttINtCFJQLwxgd+c/o751MwAzjyKnVdDUU2gF/tzvCiWre8Nr+gY4+xemdwc0c7t+n49aYvIf2BVdOMcDZT4JwJzPHyPh3WVIC6TsHyxyFwPFFtN1Jqsgq3bpjP8gMrshoFntVbhawKTyJ6cYTbCd8EDCPdKHo/bingEEdzCaaCROe5e/iPzCg6YWWbCPutii9qJjWGqxSKYc6hmEdFtcO0HwHWL9KFTAzE9ak5EWmOnTnvYbh82L9jyGDTp6O0g2+an9TEZcMrc7rP0qdvDtcBQxsqT6rj6mB0MWEQGbAg6gHERkNR/SY33/4NgTGbf+sv2hhgHv/wa2gI3u+y5EuoXgciEYLF+PwFntTnfku39bAOV9Eq+kWcq6rc9VWeyydJMkQBxJskmzfXGoLnW7CHALj8lG/UaQCeTJof16uANEUJZ5nsZxHNZjfdM0z4sWApaXVCiCOKbId8pS5dtK0JDhUbF0tOl+UxLO30fd+uvDi8Rgw1pjO8jtrqobNNHIEd279Pfj3WgcAxqbUDF22lm4ZNXfc8Td7FevI9Hq8XE72OUV4b45xigF77mG7FB3PTUg6t/o0/telxkgbtmCbdAIBd2pUcgjwSvZHS7tsx8XScPSBy/L2D+ac5kl1NBE76A9YpQGSa8Yqvh16a72HpA1JXWWJkL4HZIB4kvqheQuMEgQR3JgnMp9eThW1em8HqdTVR3KPKdc03fB+/i1x7yNQXiPE2PcSm7FviNUuYKjIUANFiUI2R0J/3EMIt2RtUOSn0WLXbFO1RD/umGJ/tYxuXaq4pPcg7Vwz4F3fxVGuLspdudoU8cnlTEBvgigOqfT87h4DNLP5CGAJ+c2Prki4gN4iRN/SO7gER/tQMDppbgxwieSV4Tv8iMk3AAbEgWoNp8EmDlds5e0SIb43tUh6RR4pp1om6ddM+VdCxhTEem00ygI8IGS5jdwW2IXf9QnJHManQxIrl58aOcmrmRQZ6oi2wCgDxh0FpM4rRcutgqYHbxa0f3gpYQiHN/3kYmzHR7KogYmMvVcGn4gCfXZFrHJFu2xBgoNTUYqGRMsJIL1fh9ik3Qm2yME+5lP2umJYcwjJFW0kisIZOUI1EMgwNt/SQIEqNPi1ENgifkiENCK0Z2Q3v8Q+UHCZuy/XLwhHwsEVB1p+VMu3H/0MT62NlA+cxedm0IG9RUxzvtBY9PVjX/XKwkEuIxr3mcDSnHAT5DcSabNMYQlRYZyLJmCeO32EB+pKHp2MTPaPPO324kY4EjXTZ8g387EZE286bomb7nHum4CvF3jNIj3+5mq2s9mIVB0oCDU5/FUYSWiD3qtl5bOZ0XEKrrNuVsLwzinmzj9TvhBr9I9HQdujfy1LnoayoVP0rimU3wCAxGy55WvROnaW6e1iziQH0ZrIj1TQvEKi9v9QkJ90Y7jySc0ke6jqQJZPNly8kkDkjGJUCqjCb+dUIPa1g4zRhCYKgbSy9kK97u+R73vM4BMU4eBqZuarhlQe2opaAE0mzVJ5Env+8SXHuRJoO2fhq6YiZ0jQSUPplqnOrE38GMrGT0boRwRb8YnBA6E/GG4jrlVIBpTmMJvvXVxc8v1GL2uohAvvKkNigyP428TpN3AwZvbHMjKTyhrZ/cDCR1v8z1gNJuy/bLasF9zgJhEdygfbsUGoTpQHtsu0jMeVR3ufXOuqkvzQnE9ZL4BZi0qLMhChGB635vbX0cDyqIS2eYHaxbSLfmmDBPx4K45qDSBiPZMvKZlgrC297G8vyclyJIgaq7iyZ1dHE0V8/E0i0/wy/KX0JrPQ29HER6L+W5iOgtj+7WoxGpNirQQSukoWob/m9occ+q4JsP8vzekhUqqeGi+wCH03tocdVKnyf1YDPmmV4Vn8ZBKdhuYYm3L3Bkr9rSpR7qx5tc1I0oCK5DfPvRnPOK+E7+SlLdnP37H4d3V5VapH2QKalpe2+f783ndLzmo+DNZRZBsEkBAfsjAG0fcv0khiIAB/IxAFIT/B+ZQVlA4INQLAADwNQCdASrAAMAAPm0skUW/v6GYOn4p+AbEtjcFbDsd8UZYAvwDOzNgP0AsQH4AfoB/gNUA/AD9ALT/yQD8ALdKHX8z1glou/f2n9yfazq/9V/pv63/t/NTVZ51Hk37h/1f7z7YP7v/wPYH+rf+f7gH6o9ID+3+gD9rv3E96T/U/sz7gv7z6gH9T/53rbf8H2BPQA/Xj03v3p+Cj+w/9790v//71n/59gD/0eoB/2erv92/QD8AP0A+vX3DvjqYen/c36+PHU3mRmV25yJ2p0puP+T3B3GYdaBJFIQlBsnSrCmJ2wgjVeQcpvfPoYshEb0MFS6QVIZjx9IUQ7/KKeYid0QGX2UstEf4dKH55LRebOnI+Yx8rEo7JZwWIZe0yPVuCAO0CjFYH6KStSzeJI3JOMnEl/PSKk3DoTxwvO51NQogBXdGLPalpkizMQLEVo11myneJjV+l/5h7SQqHUhKus+wMfmyAeJq7xobn+OZiBWwxWNoQafy3qCXHbfZd0WcDr4fUI2MbQWLe9MJq6JbLfWBfJjarDqrbRzyKVksDNkwuwbbBXZVwmE3f6v2TuUXh9Nk9NeYAAD+3CwHGYbn/+inf1w7hNJPRu8x/QyeUSNWZxg5s/eAtClOXj9p4zk0AyU4Ny25r2lVX3peagtvwBusM5CnufgFCAIcWEi7F3WRo4ccciMC/WuroZmur/wrxifcAA2qGo4Bmysk6iIoaaydSPS+VWQIAVY+6byE5/edki2HKcmwGWbJJBTFCUfHmCbT0kpFncEqze+70FNagPQ+38CJNJMXnVWSNls2DAvbfBbW3IJnzD0iK4FD4IYKL2UQNQnUJcG8CYoOOtoZF0E6nqKdyfjvILNMnlFqNaMts/e7dChO7qdgTYtjMVcLHzk5dEhCMe6gr5cBxNkPkF9ikbJqlLcsnJeVzWTnlKM2c6nqfQmxwwWrzTcqeHSP73OUFy04hB2m0/SMNeIpL2yU5jIySXM4cEqAoSs/nHjOIoTcPjRe+A4BAuefKjDY9ntCWgLkXHUh0o65Jrl/QQ7eEQOFvcgEAf4qWsTRzFSALGk8m7qKKbUZr7k1dBa9pokX6YEC7TI7+vF5xFdyWcanWXMUL0CSD7cXeXZe3PHyI5s9O+GScYQplx9M11JVKt0pNvjQSXjNa5fCuHh/TZIU5VI1/oOLp3UI7fox/oncxpxz56PBwAuAj5U9MiGX18ZkOzS404zw8g3vBw3xGPAAMgjuKTZvgGc2brLFLOMD165ysajuBfhSroSnumsxeKMhyd7tL9DlulCNm9XV6h0gitIMZz+dDL6Qc16mPde6AVppm1qEko70AJR3f9iVdw5g+TKTYUDNkuwNWIS6YWIx+OitvGOfbbDXmq41sMVAjF7I1fMHAG8Mg4PpK0Ey4M0YBSiDiJ5FZeGWzcYsiU2CIpLt/lLLicgvi/Z7eaanNuIF1jKh5blg7Mot2SJHVBgUgA+/wBtlPCPSOB5TEixStXVdSFDJ5ZM7ZLXdleDCdEaUOUsAo1hODIBXWYUJJAxMqZLVoy8sdE1J5GvIe2pZAAN3qbVabwZH4ZKhuTsgTRuqyNzBMxNACAfc4WN7EskXCzEHYXZ74zPlrIws3PhG8J1HDvT3T7g0WokAiQ5DSUFFgKJ4jiu9dPhOLWMF/ulzZUj5Xl9Jfv0XCg4FrGBl5DtewSIh2SzSFUtLhSna52+3g6OjWKss9sbieOcJHHVIFIVH7Ia1+SmEnke7A2r0DLBm6tKhbpVgKUPSTbE39QK1BUFUAe/EP7bdrVsmNXy+XbTz05UAuGVWA/DZzuuL2csd0hFD7NLg4o7Z2TAJ7LdhibiZoVALgNaEWVAMcceM6RFE8cYj3YqYLsvgD5YWI74zhFocF4UXd8cxE+BYG+ZkZz4CWNFK+45P/gsFYELz124u+N+Dkgy0giq7qYp7WUSYrGtOITLvKtqscYeDIwotjXK6BCMbjAiB6jEh9RSqr+JD+qd0CiGcYki1PwmyYml+I4/gSQb+aBtVur6r3dePeidobdIl3etUnW4lb5+pAKH33WgQhUMVvvTrbUM0iOrvb7eI/0CLpGy8lXDX8wOQc0sic7WESnjiHqEcw6bdWvU8EUClR+w+Z7Vtw5dZek7+lP8XygvCg9BLiU3kixp1O9eSMJIBg+i6m2TrDH0arRQIAZoKp6SKye294oBldW4DUeSXt2x1xH9Cd2lVeokmqYODveUmHc1k5j3j8FjCNatKki9bB0qRFoJ7LCYy1knUusZIKbGKxZ0ksx/2O3pyn5t3Njhe85zBRi+BoSSlH5m/kKj1v/5+O45CrrBK90i6VfMZ3VRq/HXiNmgaoqgIDs5+jnXHmeLQclwjUqEDfOERh3Tcte7fnRIGq9yq8RDKKYj1ViWMjLXyOxyaAedLAvEER83bOC6PewQ1eaZ8Rs7Bcx+Hv+kOR1qMGGca1iWqOSf0lxro08zq/Wb9wisi59VdAz7RGULaMjybNEtiyzjk12CNIoe6HyWy+7S0wlSskSBz7FVLuw7KcLlGD5cpGQxKKU2T5Df6STtv4zu5r9oH9rAEdSGXhN/70oRINqdWRZw5jrKGuULOASaVg3x06F8Pn6iFUbxFLnHrjtsTAZS3pqDsTHbp56fWKapwZldB9Axu0/84c+eSqX5YG5miR5qNKcp/PvNqLyGQXjcMRkuufpi+jIA3Km4xYm+4sj+QKmcyXYU1Z4vs/B8a5flpNMGQO3rqp07oewCN8AtOW8yxYi5O5cAv+R5Rk+LUzUG2A0sJkdbZ7VFqlCMheoAmZ8sD37NflUcal6yQHkh4iZQU7CvahwxClapkOQRAgcDcbJtV7lTZIie8j1PnI2ZhSTJybwcEjA6S5r+mgeDLAGw2dEMNeoOw1N0kF/Wpw+1GcLVXFn/tA9M4FvbmkDmDlEUmXc7/aPtjctjgxp8Rqhk0PT/5bXAMYJGy//E+KhGhLMddnSQdR5sQVhsoufm+E+rDHPZqvchu7yPo/MqMgfwhKJffZKIG9Ec8yxXhC/i1QV8YFTnOBiKG6gr/W91sfB9nuHPw0CmectJoxWs2mL9fs7EeNbbmCtiFt9M/1iBrscWgIASSvCwDVufrzXU+NCUTduad8qYBbNoVaRib0y+dKFkcc/y+IlTLZ7hn7LjQPf1YbqkOxtzXRbExl8Y8zUgdQXDSQmfrqY1T6e+1T2hAXOjaV2PUuiBsVG83UqhG4tn4yezzNwNa3HZP8yJkbc8I7LmrUy/1DSXaoyj1nN4nE9NjtNLs26wULsXQiHyDx7I457A5L/tRrBG0wj7MKnvjJlvdsi2CwXSPyAerjwQFcGbKkW87TNn9cH1t86T/5d1o/Y3CM0cul5yzUzVcIoPQQA1VJ2CiYDC9ka3iSb0v17HgHObCZg5O1lmuQhY8ruAFkS4BHBZjneiWnM/mx59qizAO5wqu/QIFmHVxLvokOY3NoazcFXiK6FGKH6mdjBaqJlVvc+zJRafh5V6ngnNSpCq2Rtq6jb+82HKgsAjtFJ3n92Oj0dDMG1AsC1/smwTgI8w0r8JFex0uMYX3O0POERYBj7UKTnDHEHA7lz3RIgLS/AIry1LILfE8rWNRdAzqLkfwN28EGGKrWbBG1t3uzHO0nrXGIx6XGe0IiML1Ww5RkZBZx546Tev+Cn0ijUhtSsWZDUChA2dARFy7smE9fYy3Bi2NoT4qp/5p/ASRkLKXYdynd3YG+Ep5fkYBYY9bM/6HoIXohmFtxaFJGgFitSa3C0rmgGj0RIQCDr2SAuffc76toCeh6CTjQ9iB8n1/Hdz4ePbjod3HbR9Z6o7Ne4pcm8I1Xe6QcEdzwl+Av2yXtFCQ21wxOzuiKt/YnhtFuAAcaP0hfXYnW0HE5wjBX8ixf8ouK3V0ymHsJbx/8uvb7kjM///6C3jnV453XI4fIkw0jhW6EpAjGO1iV+hD3ZAGUx//+gu1URBb15USXpHjudDuNu0PHHu3z9WdlKSU7s/0ZS4YqYZwWQT4jdceXIeTPi0KSAEzwAAA"
                style="border-radius: 4px;"
              />
            </div>
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              File Name:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              logo1.png
            </p>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Title:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              logo1
            </p>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Description:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              test
            </p>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Primary:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              No
            </p>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Created Time:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              13 Oct 2024 11:59
            </p>
            <hr
              class="MuiDivider-root MuiDivider-fullWidth css-ctpjxq-MuiDivider-root"
            />
            <h6
              class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
            >
              Modified Time:
            </h6>
            <p
              class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
            >
              13 Oct 2024 11:59
            </p>
          </div>
        </div>
        <div
          class="MuiDialogActions-root MuiDialogActions-spacing css-knqc4i-MuiDialogActions-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-colorPrimary css-1e6y48t-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Close
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      data-testid="sentinelEnd"
      tabindex="0"
    />
  </div>
</body>
`;
